main.tf

```diff
@@ -82,20 +85,39 @@ resource "ibm_database" "redis_database" {
       role     = (users.value.role != "" ? users.value.role : null)
     }
   }
-
-  group {
-    group_id = "member"
-    memory {
-      allocation_mb = var.memory_mb
-    }
-    disk {
-      allocation_mb = var.disk_mb
-    }
-    cpu {
-      allocation_count = var.cpu_count
+  dynamic "group" {
+    for_each = local.host_flavor_set ? [1] : []
```

Comment:
Based on postgres, this is insufficient (for all use cases). I assume redis supports (member)_host_flavor="multitenant". reference terraform-ibm-icd-postgres for implementation details. Also consider adding multitenant support via other_test.go.

Reply:
Thank you @shemau I addressed that